### ISSUE TITLE
Panic: Proc macro panics when template file is missing

### ISSUE THREAD

**Issue by wojciechkepka** [Kogia-sima/sailfish#55]
When adding a `TemplateOnce` derive to this `Index` struct and the specified template file is missing the proc-macro panics. When I correctly create a file in `templates` dir it compiles. What is weird is that I'm unable to reproduce it on a minimal example.

```rust
use crate::models::Note;
use sailfish::{RenderResult, TemplateOnce};
use serde::Serialize;

#[derive(Debug, Serialize, TemplateOnce)]
#[template(path = "index.stpl")]
pub struct Index {
    notes: Vec<Note>,
}

impl Index {
    pub fn new(notes: Vec<Note>) -> Self {
        Index { notes }
    }
}
```

```
error: proc-macro derive panicked
 --> src/web.rs:6:28
  |
6 | #[derive(Debug, Serialize, TemplateOnce)]
  |                            ^^^^^^^^^^^^
  |
  = help: message: stable/nightly mismatch
  ```
  
  I'll add that I tried compiling with both stable `1.50.0` and nightly `1.52.0`

**Comment by wojciechkepka** [comment]
Ok, so I just confirmed that compiling with:
```toml
[dependencies]
sailfish = "0.3"
```
results in above panic, but using master branch:
```toml
[dependencies]
sailfish = { git = "https://github.com/Kogia-sima/sailfish", branch = "master" }
```

gives correct error:
```
error: Template file "index.stpl" not found
 --> src/web.rs:6:19
  |
6 | #[template(path = "index.stpl")]
  |                   ^^^^^^^^^^^^

error: Template file "index.stpl" not found
 --> src/web.rs:6:19
  |
6 | #[template(path = "index.stpl")]
  |                   ^^^^^^^^^^^^

error: aborting due to previous error
```

**Comment by wojciechkepka** [comment]
Just checked and it got fixed in 5e12280df14a918493288db96371d89db0b7570b
